webapp/components/form/Select/Select.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import './Select.scss'
+import React from 'react'
+import PropTypes from 'prop-types'
+import ReactSelect from 'react-select'
+
+import { defaultComponents, baseStyles, baseTheme } from './config'
+import { adaptSelection } from './utils'
+
+export const Select = ({
+  style = {},
+  className: customClassName,
```

Comment:
any reason to rename this to customClassName ?

Reply:
not just I used to do that when I use styled-components